<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>widget</class>
 <widget class="QWidget" name="widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>404</width>
    <height>245</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Admin</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="Tab">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="Users">
      <attribute name="title">
       <string>Пользователи</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QTableWidget" name="table_Users"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QComboBox" name="searchParam"/>
           </item>
           <item>
            <widget class="QLineEdit" name="ToSearch"/>
           </item>
           <item>
            <widget class="QPushButton" name="search_butt">
             <property name="text">
              <string>Искать</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="clearSearch">
             <property name="text">
              <string>Сбросить</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QPushButton" name="addUser">
             <property name="text">
              <string>Добавить пользователя</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="deleteUser">
             <property name="text">
              <string>Удалить пользователя</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="saveUsers">
             <property name="text">
              <string>Сохранить</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="resetUsers">
             <property name="text">
              <string>Сбросить</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Data">
      <attribute name="title">
       <string>Данные</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QTableWidget" name="table_Data"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QPushButton" name="addData">
             <property name="text">
              <string>Добавить</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="deleteData">
             <property name="text">
              <string>Удалить</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="saveData">
             <property name="text">
              <string>Сохранить</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="resetData">
             <property name="text">
              <string>Сбросить</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
